src/index.html

```diff
@@ -8,6 +8,50 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div>
+        <img
+          src=".//images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        >
+      </div>
+
+      <h3 class="card__description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+
+      <p class="card__product">Product code: 195434</p>
+
+
+      <div class="card__reviews">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+        </div>
+
+        <p>Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <h5>Price:</h5>
+        <h5 class="card__value">$2,199</h5>
+      </div>
+
+      <div>
+      </div>
+
+      <a href="#" data-qa="hover">
+        <button class="card__button">
+          BUY
+        </button>
```

Comment:
button is redundant here
```suggestion
        BUY
```